RR.App.Tests/Controllers/HRIS/EmployeeSalaryDetailsControllerUnitTest.cs

```diff
@@ -0,0 +1,288 @@
+﻿//TODO: Complete Unit Tests
```

Comment:
Bruh.

Reply:
Apologies, I removed the unit tests and added them to a separate branch